### PR TITLE
Fix/nested serialization correction

### PR DESCRIPTION
**Pull request by bielu**
hey @rickbutterfield,
there is adjustments for nested grid in block list nested in Grid (yeah a lot of nesting), but now it will resolve issue with any level of nesting! related to #47
TO test:
- create document with grid,
- in grid create block which takes block list,
- in block list create component which uses grid
LMK if you need any more information on it

### CODE REVIEW

File: src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
 		<RootNamespace>Umbraco.Community.BlockPreview</RootNamespace>
 		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
 
-		<VersionPrefix>1.8.2</VersionPrefix>
+		<VersionPrefix>2.0.0</VersionPrefix>
```
Comment (rickbutterfield) [inline src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj]: Leave this set as 1.8.2 for the time being

File: src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj [hunk 2]
```diff
@@ -42,9 +42,9 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
-		<PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="[13.0.0,14.0.0)" />
-		<PackageReference Include="Umbraco.Cms.Web.Common" Version="[13.0.0,14.0.0)" />
-		<PackageReference Include="Umbraco.Cms.Web.Website" Version="[13.0.0,14.0.0)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" Version="[13.2.0,14.0.0)" />
```
Comment (rickbutterfield) [inline src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj]: Please revert lines 45-47 back to `"[13.0.0,14.0.0)"` so we provide the largest support for v13 sites

**Review by rickbutterfield — COMMENTED** [review]
Hey @bielu! This looks great and I've managed to replicate based on what you've said in a very quick demo. It looks as though we'll be able to migrate this to the v14 version fairly easily as well by using v14's `IJsonSeralizer` to replace `Newtonsoft.Json` when the time comes.

Only a few minor tweaks to make, which I've left as comments.

File: src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj (hunk 1 above)
Comment (bielu) [reply src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj]: Ah It was from my first version! where I was navigating through service to figure out if it si block. let me correct it :)

File: src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj (hunk 2 above)
Comment (bielu) [reply src/Umbraco.Community.BlockPreview/Umbraco.Community.BlockPreview.csproj]: as above

**Comment by bielu** [comment]
hey @rickbutterfield any eta when this can be merge and released? ;)

**Comment by rickbutterfield** [comment]
@bielu v1.8.3 is releasing to NuGet right now 😁

**Pull request merged**